releases/release-1.23/README.md

```diff
@@ -58,18 +58,18 @@ The 1.23 release cycle is proposed as follows:
 | Team finalized | Lead | Fri August 27, 2021 | week 1 | |
 | Production Readiness Soft Freeze | Enhancements Lead | Thu September 2, 2021 | week 2 | |
 | Start Release Notes Draft | Release Notes Lead | Tue September 7, 2021 | week 3 | |
+| 1.23.0-alpha.2 released | Branch Manager | Tue September 7, 2021 | Week 3 | |
```

Comment:
This is already overdue, let's move it to Sep 14 to relax the pressure on our side.

Reply:
Ack. Done.